AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Core/DataStoreList+LazyLoad.swift

```diff
@@ -58,21 +49,6 @@ extension List {
         }
     }
 
-    // MARK: - Synchronous API
-
-    /// Trigger `DataStore` query to initialize the collection. This function always
-    /// fetches data from the `DataStore.query`. However, consumers must be aware of
-    /// the internal behavior which relies on `DispatchSemaphore` and will block the
-    /// current `DispatchQueue` until data is ready. When operating on large result
-    /// sets, prefer using the asynchronous `load(completion:)` instead.
-    ///
-    /// - Returns: the current instance after data was loaded.
-    /// - seealso: `load(completion:)`
-    public func load() -> Self {
-        lazyLoad()
-        return self
-    }
```

Comment:
This is moved over to DataStoreList.swift, that subclasses List, to prevent API breaking change